components/error-text/src/index.js

```diff
@@ -23,5 +24,10 @@ const GErrorText = glamorous.span({
   },
 });
 
-const ErrorText = props => <GErrorText {...props} />;
+const ErrorText = ({ children }) => (<GErrorText>{children}</GErrorText>);
```

Comment:
this is no longer spreading props, so we'd lose the ability to set stuff like `id` or other html attributes on this element

we should either revert the change to this line (as the previous version would still work fine), or make this line:
`const ErrorText = ({ children, ...props }) => (<GErrorText {...props}>{children}</GErrorText>);`

Reply:
Cool thanks